src/Particle.js

```diff
@@ -716,15 +723,14 @@ class Particle {
 	 * See the API docs for details https://docs.particle.io/reference/api/#integrations-webhooks-
 	 *
 	 * @param  {Object} options Options for this API call
-	 * @param  {String} options.integrationType  The kind of external integration. One of Webhook, AzureIotHub, GoogleCloudPubSub, GoogleMaps
 	 * @param  {String} options.event            Event that triggers the integration
 	 * @params {Object} options.settings         Settings specific to that integration type
 	 * @param  {String} [options.deviceId]       Trigger integration only for this device ID or Name
 	 * @param  {String} [options.product]        Integration for this product ID or slug
 	 * @param  {String} options.auth             Access Token
-	 * @return {Promise}
+	 * @returns {Promise} A promise
 	 */
-	createIntegration({ integrationType, event, settings, deviceId, product, auth, context }) {
```

Comment:
another unused variable - flagging this one since the option was also documented